local-interchaintest/README.md

```diff
@@ -18,18 +18,42 @@ local-ic start neutron_juno --api-port 42069
 
 This will start a local environment with a Gaia chain, a Neutron (using ICS) chain and a Juno chain. The `--api-port` will expose the API on port 42069, we are using this port in our local-ic-utils crate so let's use the same to reuse some of the utils there.
 
+
+For tests that involve EVM chains, the chains and relayer are started from the test itself so no need to start them manually.
+
 ## Optimize Contracts
 
 Use CosmWasm optimizer to optimize contracts and store the results in `./artifacts`
 
 ``` bash
 just optimize
+
+## or
+./devtools/optimize.sh
 ```
 
 ## Running tests
 
+Check that the `local-ic start` command created `local-interchaintest/configs/logs.json` with the correct RPC URL mappings. This configuration file is required to run the example tests.
+
 Once you have your tests written, you can run them using the following command from the workspace directory, here I'm running the `polytone` tests that are in the `examples` folder:
 
 ```bash
 cargo run --package local-interchaintest --example polytone
 ```
+
+## Troubleshooting
```

Comment:
not sure where is the best place for something like this to live, or if it is something we can maintain, but could be a helpful resource to new users of the tools?